src/Service.Tests/SqlTests/RestApiTests/Find/DwSqlFindApiTests.cs

```diff
@@ -546,7 +545,15 @@ public class DwSqlFindApiTests : FindApiTestBase
                 $"FROM {_integrationTableName} " +
                 $"ORDER BY [publisher_id] ASC, [id] ASC " +
                 $"FOR JSON PATH, INCLUDE_NULL_VALUES"
-            }
+            },
```

Comment:
nit: there's an extra newline on L341, can get that as part of this change.

Reply:
fixed.